layouts/css/base.styl

```diff
@@ -3,9 +3,11 @@
 // Base styles
 body
     box-sizing border-box
-    font 300 20px/1.5 'Source Sans Pro', Arial, sans-serif
-    color #000
+    font 400 15px/1.5 "source-sans-pro", sans-serif
+    color #333333
     margin 0
+    -webkit-font-smoothing antialiased
+    text-rendering optimizeLegibility
```

Comment:
Yeah I didnt copy everything, was just a couple of base additions. I'll move the text rendering to headers only.